deeplake/util/transform.py

```diff
@@ -222,7 +221,7 @@ def _transform_and_append_data_slice(
 
                 write_sample_to_transform_dataset(out, transform_dataset)
 
-            except SampleAppendError as e:
```

Comment:
I am a bit confused, what is the reason for adding this exception? It will catch all exceptions now, even ctl+c, isn't?